src/metabase/pulse/render.clj

```diff
@@ -398,37 +398,40 @@
 
 (defn render-pulse-card
   "Render a single CARD for a `Pulse`. DATA is the `:data` from QP results (I think)."
```

Comment:
don't forget to update the docstring when you change the signature of a function. This should do it:

```clojure
 (defn render-pulse-card
    "Render a single CARD for a `Pulse`. RESULT is the QP results."   
```